### ISSUE TITLE
Elementor should have sidebar advance filter for product archive/ product category page

### ISSUE THREAD

**Issue by aditya58singh** [elementor/elementor#15977]
## Prerequisites
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
There are lot of paid plugin which are integrated with elementor like Jetsmartfilter. Elementor should have similar type of filter which can be customize. As elementor have their own product grid. 

It will be easy for all user to make a e-commerce website and we don't have to depends on other plugins. 

**Describe the solution you'd like**
Sidebar filter for e-commerce site which will help business owner to set things fast

**Comment by 79ho3ein** [comment]
hello @aditya58singh 
Did you mean elementor to add advanced filtering features to products like the jet smart filters plugin?
If yes, it would be really great for elementor to add elementor proprietary product filters. In that case, we will not need plugins such as jet smart filters.
**These filters include: checkbox filters, price range, visual filters (display of product colors, etc.).** 
In general, jet smart filters can be a good model for elementor in adding this feature.
I hope elementor has a program to add product filtering capabilities.

thank you

**Comment by aditya58singh** [comment]
> hello @aditya58singh
> Did you mean elementor to add advanced filtering features to products like the jet smart filters plugin?
> If yes, it would be really great for elementor to add elementor proprietary product filters. In that case, we will not need plugins such as jet smart filters.
> **These filters include: checkbox filters, price range, visual filters (display of product colors, etc.).**
> In general, jet smart filters can be a good model for elementor in adding this feature.
> I hope elementor has a program to add product filtering capabilities.
> 
> thank you

Yes, elementor to add advanced filtering features...  It will be a complete package for website development

**Comment by 79ho3ein** [comment]
excellent...
